Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/SplineDataProvider.cs

```diff
@@ -0,0 +1,270 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.DataProviders
+{
+    public class SplineDataProvider : BaseMixedRealityLineDataProvider
+    {
+        [SerializeField]
+        private MixedRealityPose[] controlPoints =
```

Comment:
tooltips

Reply:
These aren't rendered the way you think they are, and tooltips do not apply.